pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -359,6 +359,69 @@ func TestPriorityQueue_Delete(t *testing.T) {
 	}
 }
 
+func BenchmarkMoveAllToActiveOrBackoffQueue(b *testing.B) {
+	tests := []struct {
+		name    string
+		podsNum []int
```

Comment:
would be nice to see some comments for what's this slice holds, and the rational of the ratio between the pods in activeQ and unschedulableQ if there is any.

Reply:
It looks we are always expecting 2 values. Just give a variable for each instead of a slice.

Reply:
Updated.